### PR TITLE
SEP-8: update UX for the Action Required flow

### PR DESCRIPTION
**Pull request by marcelosalloum**
### Changes

* After getting the KYC approved, send the user straight to the “review & submit” modal.
* Update the “Review & Submit” modal text to “`KYC has been approved.` Please review the updated operations before submitting your SEP-8 payment.”
* Update the log message when resubmitting payment automatically.

### Preview

https://user-images.githubusercontent.com/1952597/124181606-1ce55400-da8c-11eb-8fb3-3c8adb917f13.mov

### CODE REVIEW

File: src/ducks/sep8Send.ts [hunk 1]
```diff
@@ -275,6 +275,7 @@ const sep8SendSlice = createSlice({
       state.data.sep8Step = getSep8NextStepOnSuccess({
         approvalStatus: action.payload.status,
         currentStep: state.data.sep8Step,
+        didUndergoKyc: !!state.data.actionRequiredResult.result,
```
Comment (quietbits) [inline src/ducks/sep8Send.ts:278]:
```suggestion
        didUndergoKyc: Boolean(state.data.actionRequiredResult.result),
```

File: src/helpers/getSep8NextStepOnSuccess.ts [hunk 2]
```diff
@@ -9,28 +9,54 @@ import { Sep8ApprovalStatus, Sep8Step } from "types/types.d";
...
+    [Sep8Step.SENT_ACTION_REQUIRED_FIELDS]: didUndergoKyc
+      ? nextStepAfterApprovalServer({
+          currentStep,
+          approvalStatus,
+        })
+      : Sep8Step.STARTING,
     [Sep8Step.COMPLETE]: Sep8Step.DISABLED,
   };
 
+  console.log(currentStep, "->", nextStepDict[currentStep]);
```
Comment (quietbits) [inline src/helpers/getSep8NextStepOnSuccess.ts:36]: Do we need to `console.log` here?

**Review by quietbits — APPROVED** [review]

File: src/helpers/getSep8NextStepOnSuccess.ts (hunk 2 above)
Comment (marcelosalloum) [reply src/helpers/getSep8NextStepOnSuccess.ts]: Thanks for spotting that: https://github.com/stellar/stellar-demo-wallet/pull/198/commits/422e5899ec5cc4b6e58d46397e6e3e9b61a11f59

File: src/ducks/sep8Send.ts (hunk 1 above)
Comment (marcelosalloum) [reply src/ducks/sep8Send.ts]: https://github.com/stellar/stellar-demo-wallet/pull/198/commits/422e5899ec5cc4b6e58d46397e6e3e9b61a11f59

**Pull request merged**